React/CalcApp/e2e/src/e2e/app.cy.ts

```diff
@@ -0,0 +1,13 @@
+import { getGreeting } from '../support/app.po';
+
+describe('calc-app', () => {
```

Comment:
is this necessary? we will not deal with e2e so i do not want to create confusion

Reply:
I removed the e2e folder with all its content